README.md

```diff
@@ -324,57 +324,84 @@ Your function will receive two arguments:
 - `options`: The options you passed to the main plugin (including your custom plugins).
 - `context`: The global context shared accross our plugin.
 
-
+<!-- #global-context-type -->
 ```typescript
 type GlobalContext = {
-    // Available from the initialization.
+    // Mirror of the user's config.
     auth?: {
         apiKey?: string;
     };
-    // Available from the initialization.
     // More details on the currently running bundler.
     bundler: {
         name: string;
         fullName: string; // Including its variant.
-        outDir: string;
+        outDir: string; // Output directory
+        // Added in `buildStart`.
         rawConfig?: any;
-        variant?: string; // Major version of the bundler (webpack 4, webpack 5)
+        variant: string; // Major version of the bundler (webpack 4, webpack 5)
```

Comment:
might be nice to tell the user that this will be an empty string for other bundlers (IIUC)

```ts
const variant =
        meta.framework === 'webpack' ? (meta.webpack.compiler['webpack'] ? '5' : '4') : '';
```

Reply:
Good call.